Editor/BuildManager.cs

```diff
@@ -457,25 +458,35 @@ public static BuildReport BuildSync(BuildProfile buildProfile, BuildPlayerOption
         currentProfile = buildProfile;
         currentTarget = options.target;
 
-        // Add Trimmer scripting define symbols
-        AddScriptingDefineSymbols(buildProfile, ref options);
-
-        // Run options' PrepareBuild
-        foreach (var option in GetCurrentEditProfile().OrderBy(o => o.PostprocessOrder)) {
-            if ((option.Capabilities & OptionCapabilities.ConfiguresBuild) == 0) continue;
-            var inclusion = buildProfile == null ? OptionInclusion.Remove : buildProfile.GetInclusionOf(option, options.target);
-            options = option.PrepareBuild(options, inclusion);
+        try
```

Comment:
Can you change the try/catch formatting here to follow the K&R style of the rest of Trimmer? (i.e. don't put the curly braces on separate lines.)

Reply:
Oops, I must have missed that. On it.